src/java/org/apache/cassandra/service/accord/txn/TxnCondition.java

```diff
@@ -460,6 +512,11 @@ public Value(TxnReference reference, Kind kind, ByteBuffer value, ProtocolVersio
             this.version = version;
         }
 
+        public static EnumSet<Kind> supported()
+        {
+            return EnumSet.copyOf(KINDS);
```

Comment:
nit: I guess either mark this `@VisibleForTesting` or just use `Sets.immutableEnumSet()` for the field?